src/ProjectFileTools/GoToDefinition/GoToDefinitionController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using EnvDTE;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using ProjectFileTools.MSBuild;
+
+namespace ProjectFileTools
+{
+    internal class GotoDefinitionController : IOleCommandTarget
+    {
+        private readonly Guid _vSStd97CmdIDGuid;
+        private readonly WorkspaceManager _workspaceManager;
+
+        internal GotoDefinitionController(IWpfTextView textview, WorkspaceManager workspaceManager)
+        {
+            TextView = textview;
+            _vSStd97CmdIDGuid = new Guid("5EFC7975-14BC-11CF-9B2B-00AA00573819");
```

Comment:
We can probably use `VSConstants.GUID_VSStandardCommandSet97` instead of re-specifying the GUID & storing it in a field